File: app/models/Instruction.scala
package models

import javax.persistence._
import scala.beans.BeanProperty

@Entity
@Table(name="instruction")
class Instruction(instructionDesc : String){

  @Id
  @GeneratedValue(strategy = GenerationType.SEQUENCE)
  @Column(name="id", nullable= false)
  @BeanProperty
  var id: Int = _

  //@ManyToOne
  //var recipe: Recipe

  @Column(name="instruction", nullable=false)
  @BeanProperty
  var instruction: String = instructionDesc


}

File: app/models/Instruction.java
package models;

import javax.persistence.*;

import com.avaje.ebean.Model;

@Entity
@Table(name="instruction")
public class Instruction extends Model{

  @Id @GeneratedValue
  //@Column(name="id", nullable= false)
  public Long id;

  @ManyToOne
  public Recipe recipe;

  @Column(name="instruction", nullable=false)
  public String instruction;

  @OneToMany
  public Ingredient ingredient;
  
}